nodejs/week2/warmup/index.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+
+
+app.get("/numbers/add", (req,res)=>{
+    const number1 = parseInt(req.query.first);
+    const number2 = parseInt(req.query.second);
+    if(req.query.first && req.query.second){
+        if (!isNaN(number1) && !isNaN(number2)){
+            const sum = number1 + number2 ;
+            res.send(`Sum of ${number1} and ${number2} is ${sum}`)
+            }
+        else{
+            res.send(`Please write numbers on the query`)
+        }
+    }
+    else {
+        res.send(`Please write a query in this format numbers/add?first=11&second=12`)
+    }
+
+});
+
+app.get("/numbers/multiply/:first/:second", (req,res)=>{
+    const number1 = parseInt(req.params.first);
+    const number2 = parseInt(req.params.second);
+    if (req.params.first && req.params.second){
+        if(!isNaN(number1) && !isNaN(number2)){
+            const multiply = number1 * number2 ;
+            res.send(`Multiply of ${number1} to ${number2} is ${multiply} `)
+        }
+        else{
+            res.send(`Please write numbers on the query`)
+        }
+    }
+    else{
+        res.send(`Please write a query in this format /numbers/multiply/11/12`)
+    }
+});
```

Comment:
Try to see if you can identify all the code that these two routes share / have in common. If you'd like to, try to extract that code into reusable functions.

Reply:
Thanks @khjcph for this wise comment, I will work on it.